data/simulation_config.json

```diff
@@ -34,6 +34,6 @@
       }
     }
   },
-  "config_output_path": "./config_output.json",
-  "result_output_path": "./result_output.json"
+  "config_output_path": "../data/simulation_config.json",
+  "result_output_path": "..data/results/"
```

Comment:
If it is going to be a directory, I'd call it `results_output_dir`.
Additionally, we have to be careful: maybe the user didn't add the last `/`, this is a common issue when dealing with directories.

Reply:
I agree that using directories may be not ideal. If we want to use path, I think we need two result output paths as follows:
"result_output_json_path": "../data/results/simulation_results.json
"result_output_messages_path": "../data/results/simulation_results.json

Could you confirm which .json file would have to be used for the conversion? Is it actually "config_output_path"? If it is, I'll just do something like that:

"config_output_path": "../data/results/simulation_config.json
"result_output_path": "../data/results/simulation_results.txt

Reply:
Good question... Maybe @staubibr can help

Reply:
The config output path is used by Cadmium to run the simulation.
The result output path is where the result of Cadmium should be placed.
I think that the viewer only uses the output messages, not the state

Reply:
The viewer uses a .json file and the output messages (not the states). @staubibr Which .json file is used for the viewer?